lib/MusicBrainz/Server/Entity/CoreEntity.pm

```diff
@@ -35,7 +35,11 @@ has 'unaccented_name' => (
 around TO_JSON => sub {
     my ($orig, $self) = @_;
 
-    return {%{ $self->$orig }, gid => $self->gid};
+    my $json = $self->$orig;
+    $json->{gid} = $self->gid;
+    $json->{name} = $self->name;
+    $json->{unaccented_name} = $self->unaccented_name;
+    return $json;
```

Comment:
Can we make this and the other subs like so, just to maintain consistency?
```perl
around TO_JSON => sub {
    my ($orig, $self) = @_;

    return {
        %{$self->$orig},
        $json->{gid} = $self->gid;
        $json->{name} = $self->name;
        $json->{unaccented_name} = $self->unaccented_name;
    };
};
```

Reply:
Okay. I only used the current method because it's faster (about 2.7x with perl 5.26 on my macbook; it doesn't have to allocate a new hash and then merge the hashes) and these `TO_JSON` methods have to be called hundreds or thousands of times on each page load. But I didn't do a proper benchmark so it may be negligible once you take the slow Moose method calls into account.

Reply:
I don't mind either actually. I just saw a lot of the code wrapping the TO_JSON as I quoted, so thought I'd point it out. If the PR version is indeed faster then maybe we'd want to keep that and switch newer code to it too.

Reply:
Well, I guess I'll keep what I had since I'm lazy. :) But it was probably premature optimization. Moose is slow, and the slowness of the `around` modifiers probably dwarfs the hash allocations.